packages/nitro-protocol/contracts/AssetHolder.sol

```diff
@@ -1,7 +1,7 @@
 pragma solidity ^0.5.11;
 pragma experimental ABIEncoderV2;
 import './Outcome.sol';
-import 'openzeppelin-solidity/contracts/math/SafeMath.sol';
+import './openzeppelin/math/SafeMath.sol';
```

Comment:
Why make this local?

Reply:
This was one strategy to get around the limitations of `solidity-docgen`.

> Downsides to `solidity-docgen`: 
- does not currently support contracts that [have external imports](https://github.com/OpenZeppelin/solidity-docgen/issues/24)

It uses `solc`, which does not allow imports from contracts outside of the directory of the importing contract.